.ci.yaml

```diff
@@ -12,6 +12,15 @@ platform_properties:
   linux:
     properties:
       os: Linux
+  linux_web:
+    properties:
+      dependencies: >-
+        [
+          {"dependency": "chrome_and_driver", "version": "version:114.0"}
+        ]
+      os: Ubuntu
+      cores: "8"
```

Comment:
Out of curiosity, what's the default? I'm cargo-culting this, and don't know if we actually want it.

Reply:
There is no default, but different options like cores: 32. Unless the test needs a higher number of cores, we should use `8` as 32 cores bots are limited.